scripts/load-climate-baseline-values.py

```diff
@@ -19,6 +19,7 @@
     parser.add_argument("-d", "--dsn", help="Database DSN in which to create new network")
     parser.add_argument("-v", "--variable", help="Name of variable to be loaded")
     parser.add_argument("-f", "--file", help="Path of file containing climate baseline values to be loaded")
+    parser.add_argument("-e", "--exclude", help="Path of file containing native ids of stations to be excluded from loading, one per line")
```

Comment:
Good way to deal with this problem, even if we don't necessarily have to use it at first.